send/atlassian_mu_process.pl

```diff
@@ -437,7 +461,15 @@ sub updateUser {
 	my $userInfo = shift;
 	my $userId = shift;
 	my $statusActive = shift;
-	if ($DEBUG) {print $statusActive ? "Updating user $userName.\n" : "Deactivating user $userName.\n" ;}
+	my $prefixUpdate = shift;
```

Comment:
not sure if it's part of the temporary solution, but i think this flag is not needed, as it can be resolved by removing the prefix only if it exists, but it's just a cosmetic detail. once the temporary code is removed, it's value will always be false

Reply:
I did it mainly for debugging purposes, and since it felt wrong that the update counter increased for users updated with the prefix, but will remove